### PR TITLE
update: go version from 1.16 to 1.19 to support helm dependencies

### PR DESCRIPTION
**Pull request by Divlik**

### CODE REVIEW

**Review by myungjin — COMMENTED** [review]
don't forget to add any small description on your commit. left one clarification question.

File: go.mod [hunk 1]
```diff
@@ -17,3 +17,132 @@ require (
 	gopkg.in/yaml.v3 v3.0.1
 	helm.sh/helm/v3 v3.8.1
 )
+
+require (
```
[the remark this answers is not in the document]
Comment (Divlik) [reply go.mod:21]: If a module specifies go 1.17 or higher, the module graph includes only the immediate dependencies of other go 1.17 modules, not their full transitive dependencies

[the remark this answers is not in the document]
Comment (Divlik) [reply go.mod:21]: if a module specifies go 1.17 or higher in its go.mod file, its go.mod file now contains an explicit require directive for every module that provides a transitively-imported package

[the remark this answers is not in the document]
Comment (Divlik) [reply go.mod:21]: Previously we are using go 1.16 so we don't have direct and indirect.but go 1.17 or higher it is required 2 different section for direct and indirect

**Pull request closed without merging**